temporal-spring-boot-autoconfigure-alpha/src/main/java/io/temporal/spring/boot/autoconfigure/RootNamespaceAutoConfiguration.java

```diff
@@ -113,6 +114,11 @@ public WorkflowClient client(ClientTemplate clientTemplate) {
     return clientTemplate.getWorkflowClient();
   }
 
+  @Bean(name = "temporalWorkflowScheduleClient")
+  public ScheduleClient scheduleClient(ClientTemplate clientTemplate) {
```

Comment:
Should `clientTemplate` be altered to 1) add `getScheduleClient()`, and 2) accept an injected `TemporalOptionsCustomizer<ScheduleClientOptions.Builder>`?

Reply:
good point, will look

Reply:
updated